databricks-sdk-java/src/test/java/com/databricks/sdk/DatabricksAuthLoadTest.java

```diff
@@ -0,0 +1,73 @@
+// These are auto-generated tests for Unified Authentication
```

Comment:
This is not autogenerated?

Reply:
https://github.com/databricks/databricks-sdk-java/pull/355/commits/68ae507c8f0888d60d5bcac44c809413aa5dec18